backend/apps/bedrooms/tests/test_bedroom_detail.py

```
@@ -0,0 +1,156 @@
+    def setup_domain(cls, domain):
+        domain.domain = cls.get_test_tenant_domain()
+        domain.is_primary = True
+
+    def setUp(self):
+        super().setUp()
+        user_model = get_user_model()
+
+        self.admin_user = user_model.objects.create_user(
+            username="admin-detail",
+            email="admin@bedroomdetail.test",
+            password="demo1234",  # NOSONAR
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "password"

(S106)